### ISSUE TITLE
./altv-server: Permission denied

### ISSUE THREAD

**Issue by FDiskas** [eisengrind/docker-altv-server#20]
I don't know the reason but it stopped working. I get error
```
/root/entrypoint.sh: line 163: ./altv-server: Permission denied
```
To fix such an issue
I created my own docker file
```
FROM eisengrind/altv-server:release

COPY . /opt/altv/

RUN chmod +x /opt/altv/altv-server
```

**Comment by pixlcrashr** [comment]
> I don't know the reason but it stopped working. I get error
> 
> ```
> /root/entrypoint.sh: line 163: ./altv-server: Permission denied
> ```
> 
> To fix such an issue I created my own docker file
> 
> ```
> FROM eisengrind/altv-server:release
> 
> COPY . /opt/altv/
> 
> RUN chmod +x /opt/altv/altv-server
> ```

Thanks for the heads up.
Idk why that happens since https://github.com/eisengrind/docker-altv-server/blob/master/Dockerfile#L17 defines altv-server to be exectuable.

I'll get into it tomorrow.

**Comment by pixlcrashr** [comment]
@FDiskas Right, so, first of all I am sorry that I took that long to look over the issue. I forgot about this error (my bad).

Now, 'thing is that I am not able to reproduce this error anyway - this may be because of recent updates of the altv-server and the image was rebuilt because of that.

Does the error still occurs at your end?

*Note: I also updated the repo such that the libnode version can be edited outside of the Dockerfile - which fixes the last few dev builds where libnode was updated.*
